test/commands/remove/tag_spec.rb

```diff
@@ -7,6 +7,14 @@
 
   let(:content) { CONTENT }
 
+  describe "when friend name and tag are blank" do
+    subject { run_cmd("remove tag") }
```

Comment:
Would you mind changing this to set `friend_name` and `tag` to `nil` or `""` instead of overwriting the `subject`? Or does that not work?